src/icons/sto-color-filled.svg

```diff
@@ -0,0 +1,9 @@
+<svg viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M14.9565 2.38057C12.8846 3.59413 8.96327 5.82608 5.2256 6.27927C4.61905 6.35281 4.10692 6.80345 4.06261 7.41283C3.62531 13.427 5.28841 24.7506 15.1559 29.699C15.4409 29.8419 15.7745 29.8649 16.0703 29.7458C18.849 28.6263 22.9939 25.3948 25.3345 19.192L20.7061 17.2523C20.6786 17.2408 20.6519 17.2281 20.6261 17.2143C20.0172 18.2022 19.0888 19.0037 17.9192 19.4456C15.108 20.5078 11.968 19.0899 10.9059 16.2787C9.84375 13.4675 11.2617 10.3275 14.0729 9.26537C16.884 8.20324 20.024 9.62114 21.0861 12.4323C21.4712 13.4515 21.5303 14.5139 21.3152 15.5048C21.3513 15.515 21.3872 15.5275 21.4227 15.5423L25.9248 17.4291C26.7445 14.6448 27.2081 11.3415 27.0931 7.45896C27.0746 6.83321 26.576 6.32693 25.9609 6.21063C23.5629 5.75729 19.5238 4.34724 16.2474 2.39542C15.8498 2.15854 15.3559 2.14666 14.9565 2.38057ZM14.7281 10.9998C12.8748 11.7 11.9401 13.7701 12.6403 15.6234C13.3405 17.4767 15.4106 18.4114 17.2639 17.7112C19.1172 17.011 20.052 14.9409 19.3517 13.0876C18.6515 11.2343 16.5815 10.2996 14.7281 10.9998Z" fill="url(#paint0_linear_20638_160552)"/>
+<defs>
+<linearGradient id="paint0_linear_20638_160552" x1="7.81315" y1="5.4046" x2="23.6105" y2="26.1046" gradientUnits="userSpaceOnUse">
```

Comment:
linear gradients have a known issue where don't render correctly in Safari. Please test once.